app/views/curation_concerns/base/_form_relationships.html.erb

```diff
@@ -1,6 +1,15 @@
-<h3><%= t("sufia.#{controller_name}.#{action_name}.in_collections") %></h3>
-<div id="collection-widget">
+<%= render 'form_in_works', f: f %>
+<h3>This Work in Collections</h3>
```

Comment:
Why aren't we using `t("sufia.#{controller_name}.#{action_name}.in_collections")` anymore?